go/test/endtoend/cluster/vtctlclient_process.go

```diff
@@ -96,3 +96,18 @@ func VtctlClientProcessInstance(hostname string, grpcPort int, tmpDirectory stri
 	}
 	return vtctlclient
 }
+
+// VtGateSplitQuery applies vitess schema (JSON format) to the keyspace
+func (vtctlclient *VtctlClientProcess) VtGateSplitQuery(keyspace string, sql string, splitCount int) (string, error) {
+	tmpProcess := exec.Command(
+		vtctlclient.Binary,
+		"-server", vtctlclient.Server,
```

Comment:
Looks like this is being run against vtctld, it needs to be run against vtgate.
When I tried running locally, the command being run is:
```
vtctlclient -server localhost:22400 VtGateSplitQuery -keyspace ks -split_count 2 "select id, name from user"
```
```
vtctld.INFO:I1101 14:50:44.985899    9084 grpc_server.go:212] Listening for gRPC calls on port 22400
```
```
vtgate.INFO:I1101 14:51:12.387798   13823 grpc_server.go:212] Listening for gRPC calls on port 22420
```

Reply:
The larger question is about the functionality itself. According to comments in the original python test, this is for MapReduce functionality.
Is that something we are keeping? Is it v2 or v3?
@sougou

Reply:
You don't have to port SplitQuery. I plan to delete it when I start the v2 cleanup.

Reply:
@arindamnayak this also means that you can delete the SplitQuery portion of initial_sharding_test

Reply:
Great, I will close this PR then. 
I was about to post the question "What is the use of vtgate SplitQuery" in the channel.